### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by youcanjewett**
image

### CODE REVIEW

File: src/components/Step.jsx [hunk 1]
```diff
@@ -4,7 +4,8 @@ const Step = ({step, active, currentIndex}) => {
 
     return (
         <>
-            <div className={"step-number"}>{active ? "I'm active!" : currentIndex + 1}</div>
+            <div className={active ? "step-number active-background" : "step-number"}>{currentIndex + 1}</div>
```
Comment (youcanjewett) [inline src/components/Step.jsx]: Is there a better way to write this?

Comment (Knguy47) [reply src/components/Step.jsx]: bring in the classnames library

https://www.npmjs.com/package/classnames

File: src/styles/step.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
+.step-number {
```
Comment (Knguy47) [inline src/styles/step.css:1]: Before you get too far into the styling, one of the requirements was to use a css pre-processor. You may want to bring it in early in the project.

File: src/styles/stepTracker.css [hunk 3]
```diff
@@ -0,0 +1,10 @@
+section {
```
Comment (Knguy47) [inline src/styles/stepTracker.css]: don't declare a global style for section. use a class name for the step tracker. (Example: What if you use a section in a different component?)

**Pull request merged**